<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fun43OrderMoveClass</class>
 <widget class="QDialog" name="Fun43OrderMoveClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Fun43OrderMove</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {	
	border-image: url(:/rc/res/main_bk.png) 0 00 0 0 repeat;
}
#widget_2{ 
border-image: url(:/rc/res/main_bk.png) 50 00 0 0 repeat;
}
QPushButton {
	border: 1px solid  rgb(0, 0, 0);
	border-radius: 0px;
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.505682 rgba(35, 97, 222, 247), stop:0.539773 rgba(61, 114, 223, 247), stop:1 rgba(93, 142, 239, 238));
	color: rgb(255, 255, 255);
	font: 16px;
}
QGroupBox::title {
  color: rgb(255, 255, 255);
  subcontrol-origin: margin;
  subcontrol-position: top center;
  padding-top: 10px;  
  border-top: 1px solid rgb(255, 255, 255);
  font:bold 14px;
}
QGroupBox {
	border: 1px solid rgb(255, 255, 255); 
    border-radius: 9px;
	padding:2px; font: bold 16px;	
	background-color: rgba(199, 199, 199, 100);
}

QTableWidget::item {
   
}
QHeaderView::section {
    background-color:  rgb(221, 221, 221); 
	font: 18px;
	border: 0px solid #6c6c6c;
	height: 25px;
}

QTableWidget {
border-left: 1px solid rgb(255, 255, 255); 
border-right: 1px solid rgb(255, 255, 255); 
background-color: transparent;
color: rgb(255, 255, 255);
font: bold 14px;
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
}</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>800</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="btOK">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>确认调拨</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(0, 0, 0);
padding-left: 80px;
font: 18px;</string>
    </property>
    <property name="text">
     <string>14141</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>工单信息</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>311</width>
       <height>291</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>9</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>78</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>属性</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数值</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>工单类型</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>派工单号</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>工单号</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>产品编号</string>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>产品描述</string>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>模具编号</string>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string>机器编号</string>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string>最小吨位</string>
      </property>
     </item>
     <item row="8" column="0">
      <property name="text">
       <string>最大吨位</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="btOrderUp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-left-radius:10px;
border-top-left-radius:10px;</string>
    </property>
    <property name="text">
     <string>上一单</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>461</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>机器列表</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>461</width>
       <height>291</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>115</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>机器ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>吨位</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="btExit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btOrderNext">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-right-radius:10px;
border-top-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>下一单</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 2px;</string>
    </property>
    <property name="text">
     <string>调拨给</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
